src/ship/PlayerShipController.cpp

```diff
@@ -227,16 +579,11 @@ void PlayerShipController::StaticUpdate(const float timeStep)
 			m_mouseActive = false;
 		}
 	}
-
-	//call autopilot AI, if active (also applies to set speed and heading lock modes)
-	OS::EnableFPE();
-	m_ship->AITimeStep(timeStep);
-	OS::DisableFPE();
 }
 
-void PlayerShipController::CheckControlsLock()
+bool PlayerShipController::IsControlsLocked()
```

Comment:
Minor nitpick - the rules of English say that this function should be named `AreControlsLocked`, as Controls is a plural noun.

Reply:
Done!